File: ouweh-vdwerf/src/main/org/uva/ql/ast/expression/binary/GreaterThan.java
package main.org.uva.ql.ast.expression.binary;

import main.org.uva.ql.ast.expression.Expression;

public class GreaterThan extends BinaryOperation {

    public GreaterThan(Expression left, Expression right){
        super(left,right);
    }

    @Override
    public String toString() {
        return String.format("(%s > %s)", this.getLeft(), this.getRight());
    }
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QL/AST/Expression/GreaterThan.cs
namespace Assignment1.Model.QL.AST.Expression
{
    public class GreaterThan : Binary
    {
        public GreaterThan(IExpression left, IExpression right) : base(left, right) { }

        public override void Accept(IExpressionVisitor visitor) => visitor.Visit(this);
    }
}